#configStat.yaml#

```diff
@@ -0,0 +1,6 @@
+root_file: histograms.root
```

Comment:
Please remove this automatically created backup file.

Reply:
removed